db/dump.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE "users" (
+  "username" varchar(40) UNIQUE PRIMARY KEY,
+  "name" varchar(255),
+  "email" varchar(255) UNIQUE,
+  "friends" json,
```

Comment:
Essa parte de `friends` acredito que seria melhor ficar separado em uma tabela pivot. A ideia de deixar em um json acredito não ser tão interessante, principalmente pelo fato de se precisar manipular um item específico você precisaria lidar com as outras também, e isso seria processamento desnecessário.

Reply:
Hmmmm acha que eu deveria criar uma outra tabela como `users.relationship`?
e usar foreign key em cada username dentro dessa tabela?

Reply:
É que eu entendi seu ponto, mas pensando melhor agora é meio foda fazer isso, pensa só comigo, para o cara visualizar todos os amigos dele o banco teria que realizar uma query muito desgastante, procurando todas as linhas aonde o username bate com o do usuário, e isso para cada amigo.... aqui não, seria bem mais 'tranquilo'.

Reply:
Entendo seu ponto, mas IMHO não é tão custoso quando manipular json se for uma consulta otimizada, mas fique a vontade, sem contar que a questão de histórico é melhor se for em uma tabela.

Reply:
Hmmm, então é isso, vou fazer como pediu, preciso estudar então talves realmente tenha um ponto kkk